src/components/NewWorkflow/AddPhase/AddPhase.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import {connect} from 'react-redux';
+import AddTask from './AddTask';
+
+class AddPhase extends Component {
+
+    state={
+        displayCard: false,
+        phaseName: ''
+    }
+
+    // catalogs changes to be displayed
+    handleChange=(e)=>{
+        this.setState({phaseName: {
+            name: e.target.value,
```

Comment:
I'm not sure if i can test this myself, but does e.target.value work to set both name and id states or would you need to use typeOf?

also, just for clarification, is the id the phase number?

Reply:
oh it cut things off, if you look at line 16 below thats how it gets the id, needs to get it from parent element so a seemingly complicated way to hit it but works just fine